manager2/src/app/my-delete-confirm/my-delete-confirm.component.html

```diff
@@ -2,8 +2,12 @@
   <div *ngIf="!isDeleting" >
     <button type="button" class="btn btn-danger btn-xs" (click)="startDelete()"><i class="fa fa-trash-o"></i> Delete</button>
   </div>
-<div *ngIf="isDeleting">
-  <button type="button" class="btn btn-outline-secondary btn-xs" (click)="cancel()">Abort deletion</button>
-  <button type="button" class="btn btn-danger btn-xs"  (click)="confirm()"><i class="fa fa-trash-o"></i>Confirm</button>
+  <div *ngIf="isDeleting">
+    <label for="message" class="col-4 col-form-label"><b>Please Explain Why:</b></label>
```

Comment:
will be shown even if explainMessage is false

Reply:
https://github.com/genouest/genouestaccountmanager/pull/123/commits/3ee874225bc09c0b260da058d5d695e570298a53